pkgs/development/go-modules/generic/default.nix

```diff
@@ -55,11 +57,24 @@
 with builtins;
 
 assert goPackagePath != "" -> throw "`goPackagePath` is not needed with `buildGoModule`";
+assert (vendorSha256 == "_unset" && vendorHash == "_unset") -> throw "either `vendorSha256` or `vendorHash` is required";
+assert vendorSha256 != null && vendorSha256 != "_unset" && vendorHash != null && vendorHash != "_unset"
```

Comment:
I think this still allows for both to be set and `null`, if I'm not mistaken.

Reply:
We shouldn't allow bot to be set.